agr_indexer/src/main/java/org/alliancegenome/indexer/indexers/curation/DiseaseAnnotationCurationIndexer.java

```diff
@@ -441,8 +431,19 @@ private static String getConsolidationKey(DiseaseAnnotation da, String relation)
 	}
 
 	private static void populateGeneticModifier(DiseaseAnnotation da, DiseaseAnnotationDocument adad) {
-		if (CollectionUtils.isNotEmpty(da.getDiseaseGeneticModifiers())) {
-			List<BiologicalEntity> geneticModifiers = da.getDiseaseGeneticModifiers().stream().filter(Objects::nonNull).toList();
+		if (CollectionUtils.isNotEmpty(da.getDiseaseGeneticModifierAlleles()) ||
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.OperatorWrapCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'||' should be on a new line.
<!-- __reviewdog__:ChAxYWY2YTc3OTljYzhjMjExEgpjaGVja3N0eWxl -->